backend/migrations/2023-02-04-132532_create_plant_detail/up.sql

```diff
@@ -0,0 +1,45 @@
+-- Your SQL goes here
+CREATE TABLE plant_detail (
+  id SERIAL PRIMARY KEY,
+  binomial_name VARCHAR NOT NULL,
+  genus VARCHAR,
+  family VARCHAR,
+  edible_uses VARCHAR,
+  medicinal_uses VARCHAR,
+  material_uses_and_functions VARCHAR,
+  botanic VARCHAR,
+  propagation VARCHAR,
+  cultivation VARCHAR,
+  environment VARCHAR,
+  material_uses VARCHAR,
+  functions VARCHAR,
+  provides_forage_for VARCHAR,
+  provides_shelter_for VARCHAR,
+  hardiness_zone INTEGER,
+  heat_zone VARCHAR,
+  water VARCHAR,
+  sun VARCHAR,
+  shade VARCHAR,
+  soil_ph VARCHAR,
+  soil_texture VARCHAR,
+  soil_water_retention VARCHAR,
+  environmental_tolerances VARCHAR,
+  native_climate_zones VARCHAR,
+  adapted_climate_zones VARCHAR,
+  native_geographical_range VARCHAR,
+  native_environment VARCHAR,
+  ecosystem_niche VARCHAR,
+  root_zone_tendancy VARCHAR,
+  deciduous_or_evergreen VARCHAR,
+  herbaceous_or_woody VARCHAR,
+  life_cycle VARCHAR,
+  growth_rate VARCHAR,
+  mature_size VARCHAR,
```

Comment:
we have 2 columns now, one for height and one for width. However, the data should be normalized since there are erroneous inputs, see the file below with distinct values from all articles

[value_varchar.csv](https://github.com/ElektraInitiative/PermaplanT/files/10675111/value_varchar.csv)